web/components/nav/bottom-nav-bar.tsx

```diff
@@ -50,36 +60,21 @@ export function BottomNavBar() {
     return null
   }
 
-  const navigationOptions =
-    user === null ? signedOutNavigation : getNavigation()
+  const navigationOptions = !user ? signedOutNavigation : getNavigation(user)
```

Comment:
This line is different when `user` is `undefined`. (Which is often true on the first render.)

I think it might be better to default to the signed in navigation since that is most common.